contrib/google.golang.org/grpc/grpc_test.go

```diff
@@ -119,7 +149,16 @@ func TestStreaming(t *testing.T) {
 				assert.Equal(t, rig.port, span.Tag(ext.TargetPort),
 					"expected target host port to be set in span: %v", span)
 				fallthrough
-			case "grpc.server", "grpc.message":
+			case "grpc.server":
```

Comment:
I fail to see the difference between these two. Can we merge them back into the same switch-case?
```go
case "grpc.server", "grpc.message":
```